README_DATA.md

```diff
@@ -62,17 +66,16 @@ Some common functions between the table builder and chart builder are also store
 
```

Comment:
Hmm, not sure if maybe the file names above could be improved slightly. `rd-chart-builder.js`, `rd-chart-renderer.js`, and `rd-chart-builder-utils.js` might be more explicit? I know I was confused about what the files were meant to contain when I first came across them.

Reply:
could be but those will come during the frontend refactoring